src/niscope/examples/niscope_fetch_forever.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python
+
+import argparse
+import datetime
+import niscope
+import numpy as np
+import pprint
+import sys
+
+
+pp = pprint.PrettyPrinter(indent=4, width=80)
+
+
+# This example is based on the C FetchForever example
+def example(resource_name, channels, options, total_acquisition_time_in_seconds, voltage, sample_rate_in_hz, samples_per_fetch):
+    total_samples = int(total_acquisition_time_in_seconds * sample_rate_in_hz)
+    channel_list = channels.split(',')  # We need channels as a list
+
+    # 1. Creating numpy arrays
+    waveforms = [np.ndarray(total_samples, dtype=np.float64) for c in channel_list]
+
+    # 2. Opening session
+    with niscope.Session(resource_name=resource_name, options=options) as session:
+        # 3. Configuring
+        session.configure_horizontal_timing(min_sample_rate=sample_rate_in_hz, min_num_pts=1, ref_position=0.0, num_records=1, enforce_realtime=True)
+        session.channels[channel_list].configure_vertical(voltage, coupling=niscope.VerticalCoupling.DC, enabled=True)
+        # We configure for a software trigger and then never send it
+        session.configure_trigger_software()
+        current_pos = 0
+        # 4. initating
+        with session.initiate():
+            while current_pos < total_samples:
+                # We fetch each channel at a time so we don't have to de-interleave afterwards
+                # We do not keep the wfm_info returned from fetch_into
+                for c, wfm in zip(channel_list, waveforms):
+                    # 5. fetching
+                    session.channels[c].fetch_into(wfm[current_pos:current_pos + samples_per_fetch], relative_to=niscope.FetchRelativeTo.READ_POINTER,
+                                                   offset=0, record_number=0, num_records=1, timeout=datetime.timedelta(seconds=5.0))
+                current_pos += samples_per_fetch
+
+        # 6 fetch complete
+    # 7 session closed
```

Comment:
I think it's ok to take out most of these comments. Several are just oddly placed and several mostly paraphrase the code.
I assume the C example was written this way?

Reply:
They were originally `print()` so I could track what was going on.

Reply:
I think it's safe to delete. The code is really clean and self-explanatory.